eplaunch/interface/config.py

```diff
@@ -0,0 +1,141 @@
+from collections import deque
```

Comment:
JSON based config file management.  Pretty simple, just opening/closing/updating a JSON blob.  No need to rely on a framework-specific managed cache.  The config file is now named `.EP-Launch.json` so it doesn't get confused with the Tk EPLaunch config file.